README.md

```diff
@@ -1,2 +1,66 @@
-# andino_docs
-Manages andino ecosystem documentation
+# Andino Docs
+
+Manages andino ecosystem documentation.
+
+Visit the [website](https://andino-documentation.readthedocs.io/en/latest/) to see all th information.
+
+## Curiosities
+
+- If you want to add images, you can use the relative path directly. If you want to add videos, use always the `_static` directory, as there is where they are placed inside the `html` directory, even though you initially have it inside the `media` folder.
+- In this case, no Github Action is required for ReadTheDocs to work, we just need to setup the Webhook on the repository `Settings` to be able to build the documentation automatically every time a push is done or a pull request is merged. This is configured automatically if you are admin on the repo group. If not, refer to [this link](https://docs.readthedocs.io/en/stable/guides/setup/git-repo-manual.html).
+
+## Get README.md
+
+A bash script named `get_readme.sh` is included in the repo to retrieve the latest version of the `README.md` of andino packages to include them in the documentation. You just need to execute the script, it will clone the repo, extract the files and remove the repo:
+
+```sh
+chmod +x get_readme.sh
+./get_readme.sh
+```
+
+## Build documentation locally
+
+**NOTE**: Use a virtual environment if you don't want to keep these requirements in your machine.
+
+### Install sphinx
+- Install ``pip``
+    ```sh
+    sudo apt install python3-pip
+    ```
+- Install ``sphinx``
+    ```sh
+    pip install sphinx
```

Comment:
nit: note that you have a requirements.txt file so you can pip install from there directly.

Reply:
Oops you are totally right I worte that part before creating the `requirements.txt` and I forgot to change it , thanks!